docs/api-aggregation-setup.md

```diff
@@ -0,0 +1,183 @@
+
+# Setting up Service Catalog for API Aggregation in Kubernetes
+
+The aggregator is a server that sits in front of everything. 
+
+We need to provide a set of certificates to be provided as a
+certificate bundle to the APIService apiregistration endpoint.
+
+For development purposes, it is convenient to use the existing CA
+automatically set up by the kubernetes development environment.
+
+The aggregator is a new feature of kubernetes and is an alpha API
+running as a separate pod in kubernetes v1.6. The aggregator is
+enabled by default in kubernetes v1.7 as a beta API, and is the
+default secure endpoint directly integrated into the core kubernetes
+APIServer.
+
+For background on why we're messing with certificates and CA's at all,
+please check [the auth doc](auth.md).
+
+# Steps
+
+## Prerequisites
+You need `cfssl` tools, install with `go get -u github.com/cloudflare/cfssl/cmd/...`.
+
+## Check that aggregator is enabled
+
+`kubectl api-versions` MUST list `apiregistration.k8s.io/v1beta1` (or
+`apiregistration.k8s.io/v1alpha1` if running in kubernetes v1.6)
+
+This won't show up if you are talking to the insecure port. The
+insecure port is not behind the aggregator.
```

Comment:
I'd prefer to make this a little more concrete since it could imply that we chose to not put it behind the aggregator... how about `As of now the aggregator only supports routing of requests sent to the secure port` or something like that.  Make it clear that its not our choice/config-option - its the design of the aggregator.